src/sgx/libpoet_bridge/enclave.cpp

```diff
@@ -18,6 +18,8 @@
 #include <iostream>
 #include <sstream>
 #include <stdexcept>
+#include <iomanip>
```

Comment:
Are these include statements being used?

Reply:
these include are not used, will remove theam